apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -285,7 +282,11 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
           {!fileModeLocked && (
             <td>
               <LabelText>
-                {isTestModeResults ? <TestMode>Test</TestMode> : 'Official'}
+                {isTestModeResults ? (
+                  <Font color="warning">Test</Font>
```

Comment:
In the preview video, this text looked black. Is it meant to be orange?

Reply:
The "warning" color falls back to the default text color in medium and high-contrast modes, since we were unable to find any appropriate amber-adjacent colors that meet the >= 10:1 contrast ratio for medium contrast. (We don't yet provide a way to switch to low-contrast/dark mode in VxAdmin, but if we did, this would then show up as amber)